modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -28,11 +41,60 @@ export interface UploadResult {
 }
 
 interface EditorUpload {
+  /**
+   * Cache of blob elements created in an editor instance.
+   *
+   * @property blobCache
+   * @type Object
+   */
   blobCache: BlobCache;
+
+  /**
+   * Adds a custom filter that controls the images which are included in the scan.
+   * Images must return true on every added filter to be considered valid.
+   *
+   * @method addFilter
+   * @param {Function} filter Function which filters each image upload.
+   * @example
+   * // Filter which images are uploaded.
```

Comment:
```suggestion
   * // Filter images by their size
```

Ignore if this doesn't sound right

Reply:
That's correct, I was following the style for other examples in APIs where the comment describes the API rather than the functionality in the example.